mm/2s2h/Rando/Logic/Regions/Central.cpp

```diff
@@ -260,7 +260,8 @@ static RegisterShipInitFunc initFunc([]() {
     };
     Regions[RR_MILK_BAR] = RandoRegion{ .sceneId = SCENE_MILK_BAR,
         .checks = {
-            // TODO : Add shop checks/Circus Leader's Mask checks.
+            // TODO : Add shop checks.
+            CHECK(RC_MILK_BAR_CIRCUS_LEADER_MASK, CAN_BE_DEKU && CAN_BE_GORON && CAN_BE_ZORA && HAS_ITEM(ITEM_OCARINA_OF_TIME)),
```

Comment:
Don't we also need Romani Mask to enter the bar at the right time?

Reply:
Thats already in the entrance:
![image](https://github.com/user-attachments/assets/975dfe47-e66c-41ac-8ca1-f47e00b382cd)

Reply:
lol, I'm blind